recipe/meta.yaml

```diff
@@ -1,29 +1,32 @@
 {% set name = "hydra-core" %}
-{% set version = "1.0.6" %}
+{% set version = "1.1.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/facebookresearch/hydra/archive/v{{ version }}.tar.gz
-  sha256: fb1ad4f26332d0d88a180731204a42aed596e3edc0627b6c45cabb40fb1cf9b7
+  sha256: 5a50c0eeadcad4324a6bd895b6dcbe17068558143c8080e1cc6cdbe637c679de
 
 build:
   noarch: python
-  number: 1
-  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  script: 
+    - {{ PYTHON }} setup.py antlr
+    - {{ PYTHON }} -m pip install . -vv
 
 requirements:
   build:
     - antlr ==4.8
   host:
-    - python >=3.6,<3.10
+    - python >=3.6
     - pip
   run:
-    - python >=3.6,<3.10
-    - omegaconf >=2.0.5,<2.1
+    - python >=3.6
+    - omegaconf
```

Comment:
Please keep the same version constraints as Hydra (Any OmegaConf 2.1 version):

https://github.com/facebookresearch/hydra/blob/437e20df9f0f3cb671e117c382f2dee727afbbf2/requirements/requirements.txt#L1